### PR TITLE
[16.0] [OU-ADD] website_event_track

### PR DESCRIPTION
**Pull request by nguyenvietlam0640**

### CODE REVIEW

**Comment by legalsylvain** [comment]
/ocabot migration website_event_track

<dependency></dependency>

File: openupgrade_scripts/scripts/website_event_track/16.0.1.3/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        """,
+    )
+
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE event_track_location
+        SET sequence = 10
+        """,
+    )
```
Comment (SirAionTech) [inline openupgrade_scripts/scripts/website_event_track/16.0.1.3/pre-migration.py]: The new column should have already been created by the module and have the default value of `10`, could you please check if this is actually needed?

**Review by SirAionTech — COMMENTED** [review]
Thanks for the PR!

File: openupgrade_scripts/scripts/website_event_track/16.0.1.3/pre-migration.py (hunk 1 above)
Comment (pedrobaeza) [reply openupgrade_scripts/scripts/website_event_track/16.0.1.3/pre-migration.py]: Indeed it is that way.

**Comment by pedrobaeza** [comment]
I have pushed a rebased branch and fixing some things:

- No need to pre-create `sequence` column. It's done by the ORM.
- Translations of the fields changed in noupdate_changes.xml should be deleted.
- Some formatting on the work file.

**Pull request merged**